go.mod

```diff
@@ -11,103 +11,103 @@ replace (
 )
 
 require (
-	github.com/adrg/xdg v0.3.1
+	github.com/adrg/xdg v0.4.0
 	github.com/golang/mock v1.6.0
 	github.com/gorilla/mux v1.8.0
-	github.com/gorilla/websocket v1.4.2
-	github.com/pborman/uuid v1.2.0
+	github.com/gorilla/websocket v1.5.0
+	github.com/pborman/uuid v1.2.1
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.12.1
-	github.com/rancher/apiserver v0.0.0-20221220225852-94cba4f28cfd
+	github.com/prometheus/client_golang v1.14.0
+	github.com/rancher/apiserver v0.0.0-20221229135954-26bed53611c4
 	github.com/rancher/dynamiclistener v0.3.5
-	github.com/rancher/kubernetes-provider-detector v0.1.2
+	github.com/rancher/kubernetes-provider-detector v0.1.5
 	github.com/rancher/norman v0.0.0-20221205184727-32ef2e185b99
 	github.com/rancher/remotedialer v0.2.6-0.20220104192242-f3837f8d649a
 	github.com/rancher/wrangler v1.0.1-0.20221202234327-1cafffeaa9a1
-	github.com/sirupsen/logrus v1.8.1
-	github.com/stretchr/testify v1.8.0
-	github.com/urfave/cli v1.22.2
-	github.com/urfave/cli/v2 v2.1.1
-	golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4
-	k8s.io/api v0.25.4
-	k8s.io/apiextensions-apiserver v0.25.4
-	k8s.io/apimachinery v0.25.4
-	k8s.io/apiserver v0.25.4
+	github.com/sirupsen/logrus v1.9.0
+	github.com/stretchr/testify v1.8.1
+	github.com/urfave/cli v1.22.10
+	github.com/urfave/cli/v2 v2.23.7
+	golang.org/x/sync v0.1.0
+	k8s.io/api v0.26.0
```

Comment:
Are we ready to use the 0.26 versions of the K8s modules?

Reply:
Hmm, good question.  It slipped in with my mod updating.  I can pull it back for now if we're not sure.

Reply:
Changed to only bump rancher/apiserver